docker-compose.yml

```diff
@@ -91,7 +91,8 @@ services:
         condition: service_healthy
     platform: linux/amd64
     volumes:
-      - .:/usr/src/app
```

Comment:
This was the real source of the "node_modules state file not found" error, not the Yarn version